internal/webhooks/frrconfiguration_webhook.go

```diff
@@ -151,6 +158,21 @@ func validateConfig(frrConfig *v1beta1.FRRConfiguration) error {
 				// shouldn't happen for creates as it would be considered an update, and in any case
 				// we add the updated one at the end because for updates we don't want the old and updated resources
 				// to be considered together.
+				for _, rold := range cfg.Spec.BGP.Routers {
+					for _, rnew := range frrConfig.Spec.BGP.Routers {
+						if rold.ID != rnew.ID {
+							continue
+						}
+						for _, nold := range rold.Neighbors {
+							for _, nnew := range rnew.Neighbors {
+								// router bgp 7003' -c 'neighbor 11.11.11.1 graceful-restart'
+								if nold.ASN == nnew.ASN && nold.Address == nnew.Address && nold.EnableGracefulRestart != nnew.EnableGracefulRestart {
+									warnings = append(warnings, "Graceful restart configuration changed, reset this peer to take effect")
```

Comment:
you can continue here

Reply:
also, we don't offer a way to reset, I'd mention something like "it will be available on the next restart" too

Reply:
done